ansible/scripts/ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+roles_path = ../roles
```

Comment:
Does the ``ansible.cfg`` file has to be under ``scripts`` directory? Why not put the file into ``ansible`` directory and set  ``roles_path = roles``?